cabal/ganeti.template.cabal

```diff
@@ -1,5 +1,5 @@
 name:                ganeti
-version:             2.16
+version:             3.0
```

Comment:
Not that it matters, but I'm not sure calling it 3.0 already is best :) I would have chosen something like 2.99 for now.